Source/Core/Core/CoreTiming.cpp

```diff
@@ -479,8 +479,8 @@ void Idle()
 
 	idledCycles += DowncountToCycles(PowerPC::ppcState.downcount);
 	PowerPC::ppcState.downcount = 0;
-
-	Advance();
+	if(_M_ARM_32) //Temp Fix
```

Comment:
This should most definitely not be necessary, hence why I closed this. If there's an issue in the JIT, it's in the JIT code, not the timing or event code.